doc/getting_started/installation_linux.rst

```diff
@@ -150,6 +165,13 @@ Follow these steps to install the SDK on your Linux host system.
       export ZEPHYR_TOOLCHAIN_VARIANT=zephyr
       export ZEPHYR_SDK_INSTALL_DIR=<sdk installation directory>
 
+   On Clear Linux, the environment has default CFLAGS and CXXFLAGS already
```

Comment:
ins't worth to highlight it using `.. important::` or `..note:` ?

Or maybe make it a general recommendation for others distros too since if one set CFLAGS will also cause the same problem.